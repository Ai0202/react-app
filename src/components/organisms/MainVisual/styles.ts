import styled from "styled-components";

export const Wrapper = styled.section`
  max-width: 90%;
  margin: 0 auto;
  padding: 40px 0;
`
export const Img = styled.div`
  min-height: 50vh;
  background-image: url(../images/basketball.jpg);
  background-size: cover;
  background-repeat: no-repeat;
  position: relative;
`

export const Title = styled.h1`
  font-size: 6rem;
  color: #00011D;
  letter-spacing: 6px;
  font-family: 'Rowo Typeface';
  transform: rotate(-30deg);
  position: absolute;
  bottom: 1rem;
  right: -4rem;
`

export const SubTitle = styled.h2`
  font-size: 1.5rem;
  color: #fff;
`