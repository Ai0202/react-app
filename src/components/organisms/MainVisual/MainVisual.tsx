import React from "react";
import { makeStyles, createStyles, Theme } from '@material-ui/core';
import { Wrapper, Title, Img } from "./styles";

const useStyles = makeStyles((theme: Theme) => 
  createStyles({
    large: {
      height: theme.spacing(20),
      width: theme.spacing(20),
    },
  }),
);

export  const MainVisual: React.FC = () => {
  const classes = useStyles();
  
  return (
    <Wrapper>
      <Img>
        <Title>Kingyo Sukui</Title>
      </Img>
    </Wrapper>
  )
}