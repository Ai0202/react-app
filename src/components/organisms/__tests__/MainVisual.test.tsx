import React from "react"
import { render } from "@testing-library/react"
import { MainVisual } from "../MainVisual"

describe('MainVisual', () => {
  test('正しくレンダリングされること', () => {
    const { container } = render(<MainVisual />)

    expect(container).toMatchSnapshot()
  })
})