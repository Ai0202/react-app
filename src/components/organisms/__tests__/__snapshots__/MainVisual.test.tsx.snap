// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainVisual レンダリングの更新がないこと 1`] = `
<div>
  <section
    class="sc-AxjAm fpntmP"
  >
    <div
      class="sc-AxirZ czKQZ"
    >
      <h1
        class="sc-AxiKw fIdtxI"
      >
        Kingyo Sukui
      </h1>
    </div>
  </section>
</div>
`;

exports[`MainVisual 正しくレンダリングされること 1`] = `
<div>
  <section
    class="sc-AxjAm fpntmP"
  >
    <div
      class="sc-AxirZ czKQZ"
    >
      <h1
        class="sc-AxiKw fIdtxI"
      >
        Kingyo Sukui
      </h1>
    </div>
  </section>
</div>
`;
